_data/ja/proposals.yml

```
@@ -0,0 +1,23 @@
+title: 現在の仕様候補一覧
+description: このセクションでは、<a href="https://tc39.es/process-document/">我々のプロセス</a>においてステージ3に到達している、つまり完了が近いプロポーザルを紹介しています。
+how-to-read:
+  title: プロポーザル一覧の読み方
+  description: 各プロポーザルは他の資料へのリンクが貼ってあります。表示されているタグは以下の属性を示しています。各タグが何を意味しているかは以下の例を参照ください。全てのタグはマウスカーソルをホバーさせたら詳細が表示されます。
+  tags:
+    title: タグの種類とそれぞれの意味
+    last-presented: 最終提案日
+    last-presented-fake-date: December 2018
+    tests: テストケース
+    spec: 仕様詳細
+last-presented-alt: 直近の提案のノート
```

After reviewing it again, we may need to fix this to keep the translation consistent.

CC: @Naturalclar 

```suggestion
last-presented-alt: 直近のプロポーザルのノート
```

---

僕もコメントしようかと思ったんですけど、「プロポーザルを提案」という風に使っているみたいですね。どちらにしても、大事なのは「プロポーザル」と「その発表」を使い分けることだと思います。

---

では、この部分は"提案"を"プロポーザル"に置き換えようかと思います。(「その発表」は恐らく「提案」を意図していますよね？)

---

（たとえば「この**プロポーザル**は一年も**発表**されず、ずっと同じ段階に留まっている」という意味で。）

---

​Was "発表" used somewhere? (Maybe I'm misunderstanding something.) You used the word "段階", So is it related to the case when the stage is changed?

---

Sorry for the confusion. 😅 This PR doesn't currently use 発表 (and it's okay if we use a different word), but I mean that these notes are for each **presentation** about the proposal in a meeting, either to request stage advancement or to give a simple status update.

---

Oops! I thought the word before translation was "proposal"... I'm so sorry to complicate the discussion because of my misunderstanding!